### ISSUE TITLE
Invalid placement generated by LargestAreaFitFirstPackager

### ISSUE THREAD

**Issue by tyrcho** [skjolber/3d-bin-container-packing#433]
I've continued to run my suite of property based tests on 2.0.4 and I think I stumbled on a bug.

Sorry, I can only provide a canvas to reproduce the bug in Scala, but it should be pretty straightfoward to translate as a Java unit test.

Basicallly when asked to fill a container       with size (14, 185, 78) with 20 boxes 7 x 37 x 39 it generates a packaging of dimensions (14, 111, 156) which is too big.

```scala
 val container = Container
      .newBuilder()
      .withDescription("1")
      .withSize(14, 185, 78)
      .withEmptyWeight(0)
      .withMaxLoadWeight(100)
      .build()

    val packager = LargestAreaFitFirstPackager
      .newBuilder()
      .withContainers(container)
      .build()

    val p = packager.pack(
      Collections.singletonList(
        new StackableItem(Box.newBuilder().withId("Foot").withSize(7, 37, 39).withRotate3D().withWeight(0).build(), 20)
      )
    )

    val usedSpace: Iterable[StackPlacement] = p.getStack.getPlacements.asScala
    usedSpace.foreach(println)
//    Foot[0x0x0 6x38x36]
//    Foot[0x39x0 6x77x36]
//    Foot[0x78x0 6x116x36]
//    Foot[0x117x0 6x155x36]
//    Foot[7x0x0 13x38x36]
//    Foot[7x39x0 13x77x36]
//    Foot[7x78x0 13x116x36]
//    Foot[7x117x0 13x155x36]
//    Foot[0x0x37 6x38x73]
//    Foot[0x39x37 6x77x73]
//    Foot[0x78x37 6x116x73]
//    Foot[0x117x37 6x155x73]
//    Foot[7x0x37 13x38x73]
//    Foot[7x39x37 13x77x73]
//    Foot[7x78x37 13x116x73]
//    Foot[7x117x37 13x155x73]
//    Foot[0x0x74 6x38x110]
//    Foot[0x39x74 6x77x110]
//    Foot[0x78x74 6x116x110]
//    Foot[0x117x74 6x155x110]
//    

    def maxEnd(f: StackPlacement => Int) = usedSpace.map(f).max + 1

    val dx = maxEnd(_.getAbsoluteEndX)
    val dy = maxEnd(_.getAbsoluteEndY)
    val dz = maxEnd(_.getAbsoluteEndZ)

    println(Vector(dx, dy, dz).sorted)

    //Vector(14, 111, 156)
```

**Comment by skjolber** [comment]
![image](https://user-images.githubusercontent.com/1031478/154874921-d847fd51-cf26-4c53-bda4-2c1d0d8f553c.png)

Bug confirmed

**Comment by skjolber** [comment]
Fixed in 2.0.5
